test/idset.jl

```diff
@@ -0,0 +1,8 @@
+@testset "IDSet" begin
+    let idset = FluxRM.IDSet("0-10")
+        @test 3 ∈ idset
+        @test 11 ∉ idset
```

Comment:
TIL Julia supports `∈` and `∉`. Very cool!

Reply:
Haha yes, in most Julia editors you can use Latex abbreviations to insert unicode.